### PR TITLE
functionality set up w/ product ID hard coded, basic css done (modal windows css not set up yet)

### PR DESCRIPTION
**Pull request by spottyzot**

### CODE REVIEW

File: server/hrapi.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ module.exports = {
     if (pathVariable === '') {
       const params = new url.URLSearchParams(queryParams);
       apiLink = `https://app-hrsei-api.herokuapp.com/api/fec2/rfp/${widget}?${params}`;
-      // console.log('api: ', apiLink);
+      console.log('api: ', apiLink);
```
Comment (Aaronyabut) [inline server/hrapi.js:16]: Can you comment out console.logs

File: server/app.js [hunk 2]
```diff
@@ -38,10 +38,12 @@ app.put('/put', (req, res) => {
   let subCategory = req.body.subCategory || '';
 
 
+
   // console.log(req.body.queryParams, 'queryParams');
   // console.log(typeof req.body.queryParams);
   let queryParams = req.body.queryParams === undefined ? '' : req.body.queryParams;
 
+  console.log(pathVariable);
```
Comment (Aaronyabut) [inline server/app.js:46]: Can you comment out console.logs

**Review by aaronyabut — CHANGES_REQUESTED** [review]
Can you comment out some of the console.logs on client side, other than that everything looks good

**Comment by spottyzot** [comment]
console logs have been removed/commented!

**Review by aaronyabut — APPROVED** [review]
Looks good!

**Pull request merged**